provider/cmd/pulumi-resource-aws-apigateway/package.json

```diff
@@ -4,8 +4,8 @@
     "pulumi-resource-aws-apigateway": "bin/index.js"
   },
   "dependencies": {
-    "@pulumi/aws": "^6.0.4",
```

Comment:
Without upgrading AWS as well, the provider binary used an older pu/pu version that was creeping in through aws